src/components/LinkWithSearchParams.tsx

```diff
@@ -0,0 +1,23 @@
+import { Link, useSearch, type LinkProps } from "wouter"
+
+interface ForbiddenProperties {
+  /**
+   * @deprecated Please use `href` instead.
+   */
+  to?: never
+}
+
+type RemoveFields<Type, Fields extends keyof Type> = {
```

Comment:
Omit doesn't play well. I got the solution [from here.](https://github.com/microsoft/TypeScript/issues/31501#issuecomment-1079728677)